db/schema.rb

```diff
@@ -54,4 +66,19 @@
     t.index ["reset_password_token"], name: "index_profiles_on_reset_password_token", unique: true
   end
 
+  create_table "questions", force: :cascade do |t|
+    t.text "body"
+    t.bigint "virtualform_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["virtualform_id"], name: "index_questions_on_virtualform_id"
+  end
+
+  create_table "virtualforms", force: :cascade do |t|
```

Comment:
why is this here?  I thought we created this table in a separate pull request?

Reply:
I included it because it was more convenient for the migrations to include `virtualform_id` in the original. That way, I wouldn't have to create a new migration to add `virtualform_id`. It would've been more organized to have a separate PR, and I didn't document that in the PR. My bad!

Reply:
Hm that's odd. Not sure how that happened. 🤔  I'll hold off on merging this until I figure it out.

Reply:
@TurnerMooreDavis I originally rebased this branch off of the Virtualform branch, but GitHub is not recognizing the similarity between the two. Not sure why.

Reply:
@TurnerMooreDavis I think it's fine since it's just updating the schema.